roles/dhcp-server/defaults/main.yml

```diff
@@ -18,3 +18,8 @@ domain: video.debconf.org
 #staticips:
 #  hosts:
 #  - For the layout of the staticips.hosts variable, see the staticips role
+#
+
+default_install_dev: sda
+default_install_tasks:
+boxes: []
```

Comment:
I really want to find a way to merge staticips into this.

Reply:
agreed.
What are your thoughts on one big file of boxes vs one file per box?

Reply:
I don't thin ansible will let you have one file per box.

Reply:
I think for the PR we should ignore this.  it isn't a show stopper, it can be improved later when someone feels like it.

but to explain what I meant:  I mean what we are already doing:

carl@twist:~/src/tv/lca/video-stack-inventory/cboxs/inventory$ vim hosts carl@twist:~/src/tv/lca/video-stack-inventory/cboxs/inventory$ ls host_vars/
bb4.yml   cnt5.yml  cnt7.yml   odroid4.yml  odroid6.yml  pc8.yml
cnt4.yml  cnt6.yml  gator.yml  odroid5.yml  odroid7.yml  trist.yml

Reply:
Those are different. When ansible runs against a machine, it reads that file, but other machines don't get to see what's in those files.

Reply:
ah, got it.

looks like I already have support for defined IPs per box here:
{% if box.ip is defined %}host-record={{box.hostname}},{{box.ip}}{% endif %}

need to port the /etc/hosts stuff and /etc/network/interfaces which means looking up the machine from that array, which I don't really know how to do so not now.

for now the two lists can function as is and not interfere with each other, right?

Reply:
yeah, but you'll have to duplicate everything from the one into the other.

Reply:
I don't think you need to.  

static IPs and hostnames can still be manged using the existing files.
the boxes list can be empty and the new pxe installer should still work as expected. 

If you need to set a install drive, then you will need to duplicate the mac and hostmame for that box.

which is a little annoying but let's not make it a blocker. 

Thinking about LCA, I want a better understanding of your vision of etc/hosts and dns and whatever else I need to understand before I try to touch it.  otherwise I am fine setting up duplicate data.  That seems way safer than coding something I dont' quite understand.